README.md

```diff
@@ -40,6 +40,11 @@ To install the plugin, run:
   - `status <CR>` : Print the phase and conditions of CRs of a specific type, such as `cephobjectstore`, `cephfilesystem`, etc
   - `purge-osd <osd-id> [--force]` : Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs."
 
+- `debug <subcommand>...`           : Debug a deployment by scaling it down and creating a debug copy. This is supported for mons and OSDs only"
```

Comment:
We really need more details documented about the behavior of this command.
- Describe the scenario when it could be used
- Explain that it will scale down the existing deployment, start a new debug deployment without probes and that sleeps forever so you can connect and run ceph commands

Perhaps we could put these details in a new Examples section below. We don't need to include the details here so we can keep this section simple.

Reply:
Done